apps/charterafrica/src/payload/blocks/Hero.js

```diff
@@ -0,0 +1,176 @@
+import link from "../fields/link";
+import richText from "../fields/richText";
+import {
+  validateHexColor,
+  blendModeOptions,
+  MuiButtonColors,
+} from "../utils/colors";
+
+const Hero = {
+  slug: "hero",
+  fields: [
+    {
+      name: "slides",
+      type: "array",
+      minRows: 1,
+      fields: [
+        {
+          name: "title",
+          type: "group",
+          required: true,
+          fields: [
+            {
+              name: "color",
+              type: "text",
+              validate: validateHexColor,
+              required: true,
+              label: {
+                en: "Title Color",
+                fr: "Couleur du titre",
+                pt: "Cor do título",
```

Comment:
```suggestion
              label: {
                en: "Color",
                fr: "Couleur",
                pt: "Cor",
```

Title is already on group.